### DISCUSSION TITLE
When will CSP be ready

### DISCUSSION THREAD

**Discussion by plasid** [alpinejs/alpine#3171] [4. General]
This has been asked before, but could you guys give an indication on when the Content-Security-Policy version will be ready and Npm installable?
[https://github.com/alpinejs/alpine/pull/2619](url)

**Comment by ekwoka** [comment]
It's been stated by Caleb that it's not a priority at all.

So if you want it to happen, it would be smart to contribute to making it happen.

Livewire 3 seems the main priority, then Alpine and Alpine UI.
